host/3.0/buster/amd64/base/temp.txt

```diff
@@ -0,0 +1 @@
+ARG JAVA_VERSION=8u252
```

Comment:
remove

Reply:
Going to .gitignore all the /out/files and can update the script to delete this file at the end.

Reply:
Saw your comment with a command that could replace the need for this file. Incorporating.

Reply:
Introduced the .gitignore and removed need for this file